images/pulp_ci_centos/Containerfile

```diff
@@ -22,12 +22,12 @@ ENV S6_BEHAVIOUR_IF_STAGE2_FAILS=1
 # https://github.com/just-containers/s6-overlay/issues/467
 ENV S6_CMD_WAIT_FOR_SERVICES_MAXTIME=0
 
-RUN dnf -y install postgresql && \
-    dnf -y install postgresql-contrib && \
-    dnf -y install postgresql-server && \
-    dnf -y install postgresql-upgrade && \
-    dnf -y install nginx && \
-    dnf -y install redis && \
+RUN dnf -y install postgresql-13.14-1.el9 \
+      postgresql-contrib-13.14-1.el9 \
+      postgresql-server-13.14-1.el9 \
+      postgresql-upgrade-13.14-1.el9 \
+      nginx \
+      redis && \
```

Comment:
I am still seeing these lines in the logs:
```
2024-08-20T20:37:34.7605844Z [oneshot] postgres-init: [33m Postgresql database exists but will be upgraded from 12 to 13[0m
2024-08-20T20:37:34.7606104Z Creating directory /var/lib/pulp/scripts
2024-08-20T20:37:34.7606359Z Checking /var/lib/pulp/scripts permissions
2024-08-20T20:37:34.7607173Z [oneshot] fix-pulp-perms: [32mchown -R pulp:pulp /var/lib/pulp/scripts[0m
2024-08-20T20:37:34.7607412Z Checking /var/lib/pulp/tmp permissions
2024-08-20T20:37:34.7607967Z [oneshot] fix-pulp-perms: [32mchown -R pulp:pulp /var/lib/pulp/tmp[0m
2024-08-20T20:37:34.7608969Z s6-rc: info: service fix-pulp-perms successfully started
2024-08-20T20:37:34.7610554Z waiting for server to start....2024-08-20 20:21:18.220 UTC [125] LOG:  starting PostgreSQL 12.18 on x86_64-redhat-linux-gnu, compiled by gcc (GCC) 11.4.1 20231218 (Red Hat 11.4.1-3), 64-bit
```

Where are actually the postgresql13 installed? 🍭